subprojects/groovy-console/src/main/groovy/groovy/inspect/swingui/AstBrowser.groovy

```diff
@@ -403,9 +420,13 @@ class AstBrowser {
         def tabPane = mainSplitter.bottomComponent
         int tabCount = tabPane.getTabCount()
         for (int i = 0; i < tabCount; i++) {
-            if (bytecodeView.is(tabPane.getComponentAt(i))) {
+            def component = tabPane.getComponentAt(i);
+            if (bytecodeView.is(component)) {
                 tabPane.setTitleAt(i, getByteCodeTitle())
                 break
+            } else if (asmifierView.is(component)) {
+                tabPane.setTitleAt(i, getASMifierTitle())
+                break
```

Comment:
The `ASMifier` tab title doesn't update because of the `breaks`, if both `breaks` are removed that should fix it.